Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.cpp

```diff
@@ -230,6 +230,12 @@ std::string KeyboardMouse::GetSource() const
   return DINPUT_SOURCE_NAME;
 }
 
+// Give this device a higher priority to make sure it shows first
```

Comment:
Other platforms (XInput2::KeyboardMouse, QuartzKeyboardAndMouse, Touch, possibly more) should receive the same treatment.

Reply:
Mac and Linux Keyboard and Mouse are always first because they are not populated asynchronously, only windows (DInput/XInput) devices are. They will appear in the same order they always appeared, and the order is decided by the ControllerInterface backend "PopulateDevices()" calls.